src/Parser/XbilParser.js

```diff
@@ -29,10 +29,8 @@ export function computeMinMaxElevation(buffer, width, height, pitch) {
         const pit = y * (width || 0);
         for (let x = xs; x < xs + sizeX; x += inc) {
             const val = buffer[pit + x];
-            if (val > -10.0) {
-                max = Math.max(max, val);
-                min = Math.min(min, val);
-            }
+            max = Math.max(max, val);
```

Comment:
I think this clamp was to handle the elevation error (with huge negative elevation)

Reply:
I understand, but then the example won't work, as there is huge negative elevation (for example Marina trench at -11.000m, times the scale is way beyond the -10 authorized here.

Reply:
you could add layer option to clamp the elevation?